libs/features/pages/contact-page/src/lib/components/contact-page-form-section/contact-page-form-section.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component } from '@angular/core';
+import { FormControl,FormGroup } from '@angular/forms';
+@Component({
+  selector: 'elewa-website-contact-page-form-section',
+  templateUrl: './contact-page-form-section.component.html',
+  styleUrls: ['./contact-page-form-section.component.scss'],
+})
+export class ContactPageFormSectionComponent {
+  buttonData = {
+    text: 'Send message',
+    /** base colors on button */
+    color: '#ffffff',
+    bgColor: '#292A50',
+    iconColor: '#000000',
+    borderColor: '#000000',
+    iconBackgroundColor: '#ffffff',
+
+    /** hover effect styles on the button  */
+    hoverColor: '#000000',
+    hoverBgColor: '#ffffff',
+    hoverIconColor: '#000000',
+    hoverBorderColor: '#00000',
+    hoverIconBackgroundColor: '#000000',
+  };
+  formData!: FormGroup;
+  
+   handleSubmit(){
+    console.log(this.formData.value)
+   }
+   buildContactForm(){
+    this.formData =new FormGroup({
+      name: new FormControl(''),
+      companyName: new FormControl(''),
+      email: new FormControl(''),
+      option: new FormControl(''),
+      message: new FormControl(''),
+    })
+   }
```

Comment:
Nicely done. Just an additional change, this method can be called within an ngOnInit() hook method for the form group to be created.